test/WebSites/ModelBindingWebSite/Models/Address2.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace ModelBindingWebSite.Models
```

Comment:
Could we move this to `ViewModels` namespace so we don't have two directories in the app with models?